### PR TITLE
New package: Samsung.PlayTouchLauncher version 1.2.0.32

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112017&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.PlayTouchLauncher
+PackageVersion: 1.2.0.32
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4902668&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201204/20120404155905545/Play_Touch_Launcher_1.2.0.32.ZIP
```
Comment (Trenly) [inline manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201204/20120404155905545/Play_Touch_Launcher_1.2.0.32.ZIP
```

File: manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.PlayTouchLauncher
+PackageVersion: 1.2.0.32
```
Comment (Trenly) [inline manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.installer.yaml:5]:
```suggestion
PackageVersion: 1.2.0
```
This should match control panel. The version folder will need to be renamed after committing this change.

File: manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.PlayTouchLauncher
+PackageVersion: 1.2.0.32
```
Comment (Trenly) [inline manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.yaml:5]:
```suggestion
PackageVersion: 1.2.0
```
This should match control panel. The version folder will need to be renamed after committing this change.

File: manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4902668&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201204/20120404155905545/Play_Touch_Launcher_1.2.0.32.ZIP
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: setup.exe
+  InstallerSha256: B87EC63825B263A29B1AA10166A65160C8A7F55FEB42E2A10A380282ECB66425
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /passive
```
Comment (Trenly) [inline manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.installer.yaml:16]:
```suggestion
  NestedInstallerType: burn
  NestedInstallerFiles:
  - RelativeFilePath: setup.exe
  InstallerSha256: B87EC63825B263A29B1AA10166A65160C8A7F55FEB42E2A10A380282ECB66425
```
Installer type is burn. Using exe will cause it to throw "different install technology" error during upgrade

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
image

**Comment by denelon** [comment]
@upintheairsheep I just got back from taking some time away and was looking into the relative health of the validation pipelines and noticed a fairly large number of PRs you've been submitting recently (last few weeks). Several of them have minor inconsistencies between the package and the metadata, or they don't meet all of our policy requirements for silent installs and other small things.

I'd love to see how we could help with guidance on testing or better tooling so more of them pass on the first attempt.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**